### PR TITLE
Updates to connecting to remote Jupyter Hub

### PR DESCRIPTION
**Pull request by DonJayamanne**

### CODE REVIEW

File: src/test/datascience/ipywidgets/ipyWidgetScriptManager.vscode.common.test.ts [hunk 1]
```diff
@@ -168,7 +168,7 @@ suite('IPyWidget Script Manager @widgets', function () {
         const newFiles = (await fs.getFiles(nbExtensionsFolder)).map((item) => item.toString());
         assert.deepEqual(newFiles, files);
     });
-    test('Get a list of Widgets and script paths', async () => {
+    test.skip('Get a list of Widgets and script paths', async () => {
```
Comment (DonJayamanne) [inline src/test/datascience/ipywidgets/ipyWidgetScriptManager.vscode.common.test.ts:171]: This will be fixed in a separate PR
tracked here  https://github.com/microsoft/vscode-jupyter/issues/14193

**Review by TylerLeonhardt — APPROVED** [review]
